### ISSUE TITLE
Documentation: how to sections for site generators

### ISSUE THREAD

**Issue by estruyf** [estruyf/vscode-front-matter#160]
Create a section with docs about how to use Front Matter in combination with framework or Site Generator ...

**Comment by sid-r-singh** [comment]
Do you mean you wish to create SSG specific guides ? For example docs specific to Hugo, Jkeyll etc. ?

**Comment by estruyf** [comment]
@sid-r-singh correct, to show how it can be used in combination with these site generators. Like simple steps to get you started.

**Comment by thevinitgupta** [comment]
Hey, @estruyf I would like to work on this.

**Comment by estruyf** [comment]
@thevinitgupta all help is welcome. The starting point for this can be found here: https://github.com/FrontMatter/web-documentation-nextjs/tree/main/content/docs/static-site-generators

**Comment by thevinitgupta** [comment]
Thanks @estruyf I will start on it.

**Comment by thevinitgupta** [comment]
### Isn't this the section that is being talked about above ? 👇
![image](https://github.com/estruyf/vscode-front-matter/assets/65801700/a3825230-ccc0-444d-96f0-0aa81d048fb2)

**Comment by estruyf** [comment]
Correct, and for each SSG you would add, it would be a different file which we link to on this page.
